src/host/libpixyusb/src/pixy.cpp

```diff
@@ -90,7 +90,7 @@ extern "C"
 
     return_value = interpreter.init();
 
-    if(return_value == 0) 
+    if(return_value == 0)
     {

```

Comment:
It looks like we modified a third-party source file as our own now.  If it is indeed our own file, we should further modify to fit our coding style.

This comment applies to other files in this commit as well.

Reply:
done